test/gie/spilhaus.gie

```diff
@@ -0,0 +1,3436 @@
+
+accept   114.67954107   30.64065234
+expect   9765625   15078125
+
+accept   -144.20114979   71.68566699
+expect   -8046875   15234375
+
+accept   114.71257766   31.95711321
+expect   7578125   15234375
+
+accept   -123.29190813   63.668798
+expect   -10234375   15390625
+
+accept   115.29424195   34.50353874
+expect   5390625   15390625
+
+accept   -115.24066332   56.3185274
+expect   -12421875   15546875
+
+accept   116.99297842   38.73168243
+expect   3203125   15546875
+
+accept   -112.49873728   51.40564206
+expect   -14609375   15703125
+
+accept   120.75119895   44.9437241
+expect   1015625   15703125
+
+accept   128.33661912   52.97254946
+expect   -1171875   15859375
+
+accept   114.99188045   30.00540155
+expect   14453125   15859375
+
+accept   143.38628902   61.56286712
+expect   -3359375   16015625
+
+accept   114.98733068   30.10930789
+expect   12265625   16015625
+
+accept   171.06278983   67.42397829
+expect   -5546875   16171875
+
+accept   115.11678459   30.51890035
+expect   10078125   16171875
+
+accept   -157.5136934   66.85943577
+expect   -7578125   16328125
+
+accept   115.59004066   31.43669604
+expect   8046875   16328125
+
+accept   -141.19092067   63.39902512
+expect   -8984375   16484375
+
+accept   116.34165114   32.51381309
+expect   6640625   16484375
+
+accept   -129.22404547   58.84947523
+expect   -10546875   16640625
+
+accept   117.75729262   34.28539857
+expect   5078125   16640625
+
+------------------------------------------------------------
+# Sentitive to input parameters
+------------------------------------------------------------
+operation  +proj=spilhaus
+tolerance  1 mm
+------------------------------------------------------------
+accept   130.4   -16.2
+expect   3733410.0118   -9320.8573
+------------------------------------------------------------
+operation  +proj=spilhaus +lon_c=10.1
+tolerance  1 mm
+------------------------------------------------------------
+accept   130.4   -16.2
+expect   4343770.7991   -3701935.6242
+------------------------------------------------------------
+operation  +proj=spilhaus +lat_c=30.1
+tolerance  1 mm
+------------------------------------------------------------
+accept   130.4   -16.2
+expect   3637341.2895   -2571368.8666
+------------------------------------------------------------
+operation  +proj=spilhaus +azi=9.1
+tolerance  1 mm
+------------------------------------------------------------
+accept   130.4   -16.2
+expect   3061806.4542   -1678791.7428
+------------------------------------------------------------
+operation  +proj=spilhaus +rot=40.1
+tolerance  1 mm
+------------------------------------------------------------
+accept   130.4   -16.2
+expect   3720561.6630   309609.603620
+------------------------------------------------------------
+operation  +proj=spilhaus +k_0=0.9
+tolerance  1 mm
+------------------------------------------------------------
+accept   130.4   -16.2
+expect   3360069.0106   -8388.7716
+
```

Comment:
It would be good having a test for a sphere, to check there's no divide by zero or other similar issue when e = 0.